lib/uffizzi/clients/api/api_client.rb

```diff
@@ -63,15 +70,15 @@ def delete_project(server, project_slug)
     build_response(response)
   end
 
-  def create_credential(server, params)
-    uri = credentials_uri(server)
+  def create_credential(server, account_id, params)
+    uri = credentials_uri(server, account_id)
     response = http_client.make_post_request(uri, params)
 
     build_response(response)
   end
 
-  def update_credential(server, params, type)
-    uri = credential_uri(server, type)
+  def update_credential(server, params, account_id, type)
```

Comment:
I realize that it's not necessary but maybe we can change the order of arguments to `(server, account_id, type, params)`

Because for another similar methods we use args like this:
* delete_credential(server, account_id, credential_type)
* create_credential(server, account_id, params)